config/spring19_seminar/cola_var/cola_bert.conf

```diff
@@ -0,0 +1,37 @@
+// Import the defaults using the relative path
```

Comment:
The filename is a bit misleading—I don't see anything about cola here.

Reply:
fixed now